src/main/scala/io/iohk/ethereum/network/p2p/messages/CommonMessages.scala

```diff
@@ -111,10 +118,23 @@ object CommonMessages {
     )
 
     val FirstByteOfAddress = 12
-    val LastByteOfAddress = 32
+    val LastByteOfAddress = FirstByteOfAddress + AddressLength
```

Comment:
Moved constants to `Transaction` companion object for now but I also think it will be useful to have an `Address` class.